src/components/screens/PointOfInterest.js

```diff
@@ -138,6 +160,17 @@ export const PointOfInterest = ({ data, hideMap, navigation }) => {
           </Wrapper>
         )}
 
+        {!!businessAccount &&
+          !!crossDataTypes?.length &&
```

Comment:
do you have more complex plans in mind for the crossDataTypes? as of now we set it as a constant local variable that always has to have length `3`.

Reply:
we could map directly, right. this is a very secure check for the `crossDataTypes` to not break the app, if somebody removes the variable without recognizing by others.